.github/workflows/release-base-image.yml

```diff
@@ -0,0 +1,50 @@
+name: Release Base-* Docker image
+
+on:
+  workflow_dispatch:
+    inputs:
+      target:
+        description: "Target base image, one of: base-server, base-builder, base-ci-builder, base-admin-tools"
```

Comment:
Maybe this should be of type `choice`
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs

Reply:
cool